docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,33 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The Portal component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
```

Comment:
For most other components we describe the general use, so:
 ```suggestion
<p class="description">A portal lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
```

Reply:
I've been doing a mix of both across the Base docs—sometimes one makes more sense than the other. See: [Button](https://mui.com/base/react-button/) vs [Click-away listener](https://mui.com/base/react-click-away-listener/).

In the case of the Portal, I think it makes sense either way.